.github/scripts/test-python.sh

```diff
@@ -114,7 +114,7 @@ curl -SL -O https://github.com/k2-fsa/sherpa-onnx/releases/download/speaker-segm
 tar xvf sherpa-onnx-pyannote-segmentation-3-0.tar.bz2
 rm sherpa-onnx-pyannote-segmentation-3-0.tar.bz2
 
-curl -SL -O https://github.com/k2-fsa/sherpa-onnx/releases/download/speaker-recongition-models/3dspeaker_speech_eres2net_base_sv_zh-cn_3dspeaker_16k.onnx
```

Comment:
This one with the typo is the correct URL. Please see
https://github.com/k2-fsa/sherpa-onnx/releases/tag/speaker-recongition-models